products/akeneo-pim.md

```diff
@@ -0,0 +1,151 @@
+    latestReleaseDate: 2021-05-17
+
+-   releaseCycle: "3.1"
+    codename: "Slick Hare"
+    releaseDate: 2019-04-30
+    eol: 2019-07-20
+    latest: "3.1.18"
+    latestReleaseDate: 2019-08-06
+
+-   releaseCycle: "3.0"
+    codename: "Super-Rabbit"
+    releaseDate: 2019-02-06
+    eol: 2020-08-31
+    latest: "3.0.84"
+    latestReleaseDate: 2020-08-19
+
+# Versions 2.1 to 2.3 only contained enterprise fixes or improvements.
+
+-   releaseCycle: "2.0"
+    codename: "Rebel Rabbit"
+    releaseDate: 2017-09-28
+    eol: 2019-04-30
+    latest: "2.0.52"
+    latestReleaseDate: 2019-04-23
+
+-   releaseCycle: "1.7"
+    codename: "The Old Grey Hare"
+    releaseDate: 2017-03-14
+    eol: 2019-04-30
+    latest: "1.7.41"
+    latestReleaseDate: 2019-04-04
+
+-   releaseCycle: "1.6"
+    codename: "Wackiki Wabbit"
+    releaseDate: 2016-08-30
+    eol: 2018-03-31
+    latest: "1.6.23"
+    latestReleaseDate: 2018-04-03
+
+-   releaseCycle: "1.5"
+    codename: "Fresh Hare"
+    releaseDate: 2016-03-08
+    eol: 2017-09-30
+    link: https://help.akeneo.com/en_US/everything-you-need-to-know-about-our-pim-versions#v15
+    latest: "1.5.27"
+    latestReleaseDate: 2017-09-22
+
+-   releaseCycle: "1.4"
+    codename: "Rhapsody Rabbit"
+    releaseDate: 2015-09-23
+    eol: 2017-03-31
+    link: https://help.akeneo.com/en_US/everything-you-need-to-know-about-our-pim-versions#v14
+    latest: "1.4.28"
+    latestReleaseDate: 2016-12-13
+
+-   releaseCycle: "1.3"
+    codename: "Hare Force"
+    releaseDate: 2015-02-12
+    eol: 2016-09-30
+    link: https://help.akeneo.com/en_US/everything-you-need-to-know-about-our-pim-versions#v13
+    latest: "1.3.41"
+    latestReleaseDate: 2016-04-01
+
+-   releaseCycle: "1.2"
+    codename: "Acrobatty Bunny"
+    releaseDate: 2014-08-28
+    eol: 2015-08-18 # not documented
+    link: null
+    latest: "1.2.37"
+    latestReleaseDate: 2015-08-18
+
+-   releaseCycle: "1.1"
+    codename: "Rabbit Punch"
+    releaseDate: 2014-04-16
+    eol: 2014-07-22 # not documented
+    link: null
+    latest: "1.1.3"
+    latestReleaseDate: 2014-07-22
+
+-   releaseCycle: "1.0"
+    codename: "Hare We Go"
+    releaseDate: 2014-03-06
+    eol: 2016-03-31
+    link: https://help.akeneo.com/en_US/everything-you-need-to-know-about-our-pim-versions#v10
+    latest: "1.0.3"
+    latestReleaseDate: 2014-03-27
+
+---
+
+> [Akeneo PIM](https://www.akeneo.com/akeneo-pim-community-edition/) is a Product Information
+> Management (PIM) solution that provides a single place to collect, manage, and enrich product
+> information, create a product catalog, and distribute it to sales and eCommerce channels.
+
+{: .warning }
+> Akeneo PIM is available through [multiple offerings](https://www.akeneo.com/compare-editions/).
+> This page mainly tracks releases for the free and open-source offering named _Akeneo PIM Community
+> Edition_.
+
+Akeneo PIM does follow a documented release policy. But looking at the latest releases, it seems
```

Comment:
Suggestion: Drop the last line, and instead use `releasePolicylink`.

Rest all LGTM.

Reply:
Thanks for the review!

I have chosen to leave the last sentence (slightly modified) and set the `releasePolicylink`. I think the last sentence give an important information that must be provided in the description.